docs/get-started/quickstarts/drive-rover.md

```diff
@@ -21,73 +23,126 @@ date: "2022-12-08"
-This way, you can maximize the amount of time you have using the Viam Rover.
+{{< expand "Step 2: Install an SDK" >}}
 
-If you are running out of time during your rental, you can [extend your rover rental](/get-started/try-viam/reserve-a-rover/#extend-your-reservation) as long as there are no other reservations.
-{{< /alert >}}
+Navigate to your machine's **CONNECT** tab.
+Click on any of the listed languages and follow the instructions to install the SDK.
 
-## Connect to your rover
+To install your preferred Viam SDK on your Linux or macOS development machine or [single-board computer](/components/board/), run one of the following commands in your terminal:
 
 {{< tabs >}}
 {{% tab name="Python" %}}
 
-The easiest way to get started writing an application with Viam is to navigate to your [machine's page on the Viam app](https://app.viam.com/robots), select the **CONNECT** tab's **Code sample** page, then select **Python** and copy the boilerplate code.
+If you are using the Python SDK, [set up a virtual environment](/sdks/python/python-venv/) to package the SDK inside before running your code, avoiding conflicts with other projects or your system.
 
-{{% snippet "show-secret.md" %}}
+For macOS (both Intel `x86_64` and Apple Silicon) or Linux (`x86`, `aarch64`, `armv6l`), run the following commands:
+
+```sh {class="command-line" data-prompt="$"}
+python3 -m venv .venv
+source .venv/bin/activate
+pip install viam-sdk
+```
+
+Windows is not supported.
+If you are using Windows, use the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) and install the Python SDK using the preceding instructions for Linux.
+For other unsupported systems, see [Installing from source](https://python.viam.dev/#installing-from-source).
+
+If you intend to use the [ML (machine learning) model service](/services/ml/), use the following command instead, which installs additional required dependencies along with the Python SDK:
+
+```sh {class="command-line" data-prompt="$"}
+pip install 'viam-sdk[mlmodel]'
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+```sh {class="command-line" data-prompt="$"}
+go get go.viam.com/rdk/robot/client
+```
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+```sh {class="command-line" data-prompt="$"}
+npm install @viamrobotics/sdk
+```
+
+{{< alert title="Info" color="info" >}}
+The TypeScript SDK currently only supports building web browser apps.
+{{< /alert >}}
+
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
 
-This code snippet imports all the necessary libraries and sets up a connection with the Viam app in the cloud.
+```sh {class="command-line" data-prompt="$"}
+flutter pub add viam_sdk
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{< /expand >}}
+{{< expand "Step 3: Copy and run the sample code" >}}
+
```

Comment:
I think that with this mention of the connect tab being separated from the next bit of instructions by the tabs and all the alerts, it's really easy to get confused. Consider pasting more explicit instructions into each tab; see suggestion for one below.